android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -311,6 +311,13 @@ public void onAudioFocusChange(int focusChange) {
         Log.e(TAG, "onAudioFocusChange: focuschange: " + focusChange);
     }
 
+    public void releaseResource() {
```

Comment:
Instead of an explicit release, would it make more sense to put this in `onDestroy`?

Reply:
@slycoder CustomTwilioVideoView is a View. There is no onDestroy in its lifecycle.

Reply:
Oops, good point!